### PR TITLE
Update dependency names for static builds

### PR DESCRIPTION
**Pull request by lawruble13**
resolves #___

Summary of proposed changes:
- Changes the names of the rocBLAS and rocSOLVER dependencies when building statically
-
-

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -202,7 +202,11 @@ endif( )
 
 # Package specific CPACK vars
 if(HIP_PLATFORM STREQUAL amd)
-  rocm_package_add_dependencies(DEPENDS "rocblas >= 4.2.0" "rocsolver >= 3.26.0")
+  set(rocblas_minimum 4.2.0)
+  set(rocsolver_minimum 3.27.0)
```
Comment (TorreZuk) [inline CMakeLists.txt]: Please confirm solver bump to 3.27 for 6.2 as this is different.  This is bad if solver is incorrect in release-staging and topic

**Review by TorreZuk — APPROVED** [review]
LGTM.  @daineAMD can you do a cherry pick to release-staging once this merges?
I wrote long question on 3.27 but deleted it which latest change

File: CMakeLists.txt (hunk 1 above)
Comment (TorreZuk) [reply CMakeLists.txt]: Question answered

**Comment by daineAMD** [comment]
This change has been cherry-picked into release-staging/rocm-rel-6.2 now.

**Pull request merged**